mininet/simpletest.py

```diff
@@ -89,13 +85,22 @@ def setUp():
     h1.cmd('route add -net 20.0.0.0 netmask 255.255.255.0 dev h1-eth0')
     h2.cmd('route add -net 10.0.0.0 netmask 255.255.255.0 dev h2-eth0')
 
+    dataCollect = DataCollector()
+    # Log tcpdump on interface
+    dataCollect.tcpdump_on_intf('s1-eth1')
+
+    # Log the flow tables on the switches
+    for sw in net.switches:
+        dataCollect.log_flows_on_switch(sw)
```

Comment:
I wonder if it's possible to do this immediately after we ping? That way we can see what flows are being matched on!